File: Java/CofrinhoMain.java
public class CofrinhoMain {
    public static void main(String[] args) {
        // Cria um novo cofrinho
        Cofrinho meuCofrinho = new Cofrinho();

        // Realiza operações de depósito e saque
        meuCofrinho.deposita(100);
        meuCofrinho.saca(30);

        // Verifica e exibe o saldo atual
        int saldoAtual = meuCofrinho.saldoAtual();
        System.out.println("Saldo atual do cofrinho: " + saldoAtual + " reais");
    }
}

File: PrimeiraParte/Python/cofrinho_main.py
from cofrinho import Cofrinho

# Cria uma instância do cofrinho
meu_cofrinho = Cofrinho()

# Realiza operações de depósito e saque
meu_cofrinho.deposita(100)
meu_cofrinho.saca(30)

# Lê e exibe o saldo atual
saldo_atual = meu_cofrinho.le_saldo()
print(f"Saldo atual do cofrinho: {saldo_atual}")

